extension/src/popup/components/signTransaction/Operations.tsx

```diff
@@ -19,34 +23,59 @@ interface Path {
   issuer?: string;
 }
 
+interface PredicateSwitch {
+  name: keyof typeof CLAIM_PREDICATES;
+  value: number;
+}
+
+type PredicateValue =
+  | Array<Predicate>
+  | { high: number; low: number; unsigned: boolean; _switch?: PredicateSwitch }
+  | { _value: PredicateValue; _switch: PredicateSwitch };
+
+interface Predicate {
+  _switch: PredicateSwitch;
+  _value?: PredicateValue;
+}
+
+interface Claimant {
+  _destination: string;
+  _predicate: Predicate;
+}
+
```

Comment:
these Predicate types are a bit more typing than we actually need for this current implementation, but this is useful code that was a pain to figure out and will be beneficial in the future